test-cases/jepsen/jepsen_with_raft.yaml

```diff
@@ -20,6 +20,5 @@ jepsen_test_count: 5
 jepsen_test_run_policy: any
 bare_loaders: true
 
-append_scylla_yaml: |
-  experimental_features:
-    - raft
+experimental_features:
```

Comment:
this job need to be delete... there's no such flag anymore

Reply:
Only `experimental_features` attribute is to be deleted, or the whole test config + its dedicated jenkins job?